syntax = "proto3";

package khaleesi.core;

message SayHelloRequest {
  string name = 1;
}

message SayHelloResponse {
  string message = 1;
}

service GateKeeper {
  rpc SayHello (SayHelloRequest) returns (SayHelloResponse);
}
